### ISSUE TITLE
issue should be created based on first identifier encountered

### ISSUE THREAD

**Issue by rgalonso** [alstr/todo-to-issue-action#235]
If the `IDENTIFIERS` input is specified, this can lead to unexpected results if a comment line includes multiple identifiers. For instance, consider the following `IDENTIFIERS` value:
```
[
    {"name": "FIX", "labels": []},
    {"name": "TODO", "labels": []}
]
```
If there's a comment such as this
```
# TODO: Fix this
```
then the first identifier in the _dictionary_, **not** the on the _source line_ will be matched.

The expected result from the above should be that an issue with title `Fix this` is created. However, because `FIX` is matched first, the title is `this`.

**Comment by rgalonso** [comment]
@alstr, I do **NOT** have a solution for this one yet. I only just discovered this morning while writing a test to capture #234. My next PR will include a test which captures this though. (The test is marked as an expected failure for now.)

**Comment by alstr** [comment]
Assuming this one is still open with #234.

**Comment by rgalonso** [comment]
> Assuming this one is still open with #234.

Correct. I've only encountered this once so far in my private repos, so I haven't felt any urgency to try to fix it. It's probably not a difficult change, but I haven't investigated it.
